docs/docs/gatsby-config.md

```diff
@@ -55,41 +55,41 @@ module.exports = {
 }
 ```
 
-See more about [Plugins](/docs/plugins/) for more on utilizing plugins, and to see available official and community plugins.
+Sprawdź [Plugins](/docs/plugins/) po więcej informacji na temat ich używania oraz by znależć pluginy oficjalne, jak i te od społeczności.
 
 ## pathPrefix
 
-It's common for sites to be hosted somewhere other than the root of their domain. Say you have a Gatsby site at `example.com/blog/`. In this case, you would need a prefix (`/blog`) added to all paths on the site.
+Dość często hostuje się strony w innym miejscu, niż na roocie ich domeny. Powiedzmy że strona w Gatsby znajduje się pod adresem `example.com/blog/`. W tym celu należy dodać prefiks (`/blog`) do wszystkich ścieżek na stronie.
```

Comment:
I'm not quite sure about 'roocie'. It looks not good, unfortunately.

Reply:
Można zmienić do katalogu głównego :)